### PR TITLE
Small spelling corrections

### PR DESCRIPTION
**Pull request by Jackmin801**
Just fixing some small spelling errors since I was reading this

### CODE REVIEW

File: docs/advanced_specs.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ When both teams have submitted actions, we call `env.step(actions)` and enter th
...
 
-4. For each unit with an action provided by both teams, we save all the factory / robot actions and raise a `ValueError` if there's a formatting error. For robots specifically, if a robot has `ACTION_QUEUE_POWER_COST` or more power, it is deducted from the robots power and its action queue is replaced with the new provided one. [Processing Actions and Updating Action Queues](#environment-step---normal-phase-processing-actions-and-updating-action-queues)
+4. For each unit with an action provided by both teams, we save all the factory / robot actions and raise a `ValueError` if there's a formatting error. For robot's specifically, if a robot has `ACTION_QUEUE_POWER_COST` or more power, it is deducted from the robots power and its action queue is replaced with the new provided one. [Processing Actions and Updating Action Queues](#environment-step---normal-phase-processing-actions-and-updating-action-queues)
```
Comment (bovard) [inline docs/advanced_specs.md:66]: robots is correct here

**Review by bovard — CHANGES_REQUESTED** [review]
Thanks!

File: docs/advanced_specs.md (hunk 1 above)
Comment (Jackmin801) [reply docs/advanced_specs.md:66]: whoops, good catch

**Review by bovard — APPROVED** [review]
LGTM, thanks!!

**Pull request merged**